contracts/Crowdsale.sol

```diff
@@ -220,14 +204,12 @@ contract Crowdsale is Haltable {
     }
 
     // Check that we did not bust the cap
-    if(isBreakingCap(weiAmount, tokenAmount, weiRaised, tokensSold)) {
-      throw;
-    }
+    require(!isBreakingCap(weiAmount, tokenAmount, weiRaised, tokensSold));
 
     assignTokens(receiver, tokenAmount);
 
     // Pocket the money
-    if(!multisigWallet.send(weiAmount)) throw;
+    require(multisigWallet.send(weiAmount));
```

Comment:
We could use .transfer() instead of .send(), then require() would not be needed.

Reply:
Changed to use .transfer() as requested.